sagemaker-python-sdk/tensorflow_script_mode_quickstart/tensorflow_script_mode_quickstart.ipynb

```diff
@@ -0,0 +1,274 @@
+    "```\n",
+    "\n",
+    "Let's explain the values of **--data_dir** and **--model_dir** with more details:\n",
+    "\n",
+    "- **/opt/ml/input/data/training** is the directory inside the container where the training data is downloaded. The data is downloaded to this folder because **training** is the channel name defined in ```estimator.fit({'training': inputs})```. See [training data](https://docs.aws.amazon.com/sagemaker/latest/dg/your-algorithms-training-algo.html#your-algorithms-training-algo-running-container-trainingdata) for more information. \n",
+    "\n",
+    "- **/opt/ml/model** use this directory to save models, checkpoints, or any other data. Any data saved in this folder is saved in the S3 bucket defined for training. See [model data](https://docs.aws.amazon.com/sagemaker/latest/dg/your-algorithms-training-algo.html#your-algorithms-training-algo-envvariables) for more information.\n",
+    "\n",
+    "### Reading additional information from the container\n",
+    "\n",
+    "Often, a user script needs additional information from the container that is not available in ```hyperparameters```.\n",
+    "SageMaker containers write this information as **environment variables** that are available inside the script.\n",
+    "\n",
+    "For example, the example above can read information about the **training** channel provided in the training job request by adding the environment variable `SM_CHANNEL_TRAINING` as the default value for the `--data_dir` argument:\n",
+    "\n",
+    "```python\n",
+    "if __name__ == '__main__':\n",
+    "  parser = argparse.ArgumentParser()\n",
+    "  # reads input channels training and testing from the environment variables\n",
+    "  parser.add_argument('--data_dir', type=str, default=os.environ['SM_CHANNEL_TRAINING'])\n",
+    "```\n",
+    "\n",
+    "Script mode displays the list of available environment variables in the training logs. You can find the [entire list here](https://github.com/aws/sagemaker-containers/blob/master/README.md#environment-variables-full-specification)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Training in SageMaker"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After you test the training job locally, upload the dataset to an S3 bucket so SageMaker can access the data during training:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sagemaker\n",
+    "\n",
+    "inputs = sagemaker.Session().upload_data(path='sherlock', key_prefix='datasets/sherlock')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The returned variable inputs above is a string with a s3 location which SageMaker Tranining has permissions\n",
+    "to read data from. **It has education purposes, requiring\n",
+    " more robust solutions for larger datasets:**"
```

Comment:
I don't understand what you mean with the bolded sentence

Reply:
Removed :)